utils/script.h

```diff
@@ -29,7 +29,7 @@ struct uftrace_script_info {
 	char *name;
 	char *version;
 	bool record;
-	struct strv cmds;
+	const char *cmds;
```

Comment:
Let's not expose this to users.  Native scripts can have a separate header with your change.  Maybe argc + argv[] are more appropriate for native scripts.